src/Pretzel.Logic/Commands/CommandArgumentsAttribute.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Composition;
+
+namespace Pretzel.Logic.Commands
+{
+    [MetadataAttribute]
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+    public sealed class CommandArgumentsAttribute : ExportAttribute
+    {
+        public string CommandName { get; set; }
+
+        public CommandArgumentsAttribute() : base(typeof(ICommandParameters))
```

Comment:
That is very confusing 🤣 
I think it is better the other way around, the command know the parameters it needs

Reply:
I thought about that as well. I'll recheck why I decided against it

Reply:
Should be much better now.

Reply:
It is, thanks!